app/views/hyrax/collections/show.html.erb

```diff
@@ -1,33 +1,120 @@
-<% provide :page_title, construct_page_title(@presenter.title) %>
-<div itemscope itemtype="http://schema.org/CollectionPage" class="row collection">
-  <div class="col-sm-10 pull-right">
-    <header>
-      <%= render 'collection_title', presenter: @presenter %>
-    </header>
-    <%= render 'collection_description', presenter: @presenter %>
-
-    <% unless has_collection_search_parameters? %>
-      <%= render 'show_descriptions' %>
-    <% end %>
+<%
+   provide :page_title, construct_page_title(@presenter.title)
+
+   if banner_file(@presenter.id).blank?
+     header_class = 'hyc-generic'
+     header_data_attr = ''
+	 else
+	   header_class = 'hyc-banner'
+	   header_data_attr = "data-background=\"#{banner_file(@presenter.id)}\""
+	 end
+%>
+
+<div class="hyc-container" itemscope itemtype="http://schema.org/CollectionPage">
+	<div class="row hyc-header">
+		<div class="col-md-12">
+
+		  <header class="<%=header_class%>" <%=header_data_attr%>>
+
+				<div class="hyc-title">
+					<h1><%=@presenter.title.first%></h1>
+				  <span class="label label-success">Collection</span> <%# Ask Lynette where you can grab "User Collection" from  %>
+				  <%= @presenter.permission_badge %>
+				</div>
+
+				<% unless logo_record(@presenter.id).blank? %>
+					<div class="hyc-logos">
+					  <% logo_record(@presenter.id).each_with_index  do |lr, i| %>
+
+							<% if lr[:linkurl].blank? %>
+					      <img alt="<%= lr[:alttext] %>" src="<%= lr[:file_location] %>">
+							<% else %>
+					      <a href="<%= lr[:linkurl] %>">
+					        <img alt="<%= lr[:alttext] %>" src="<%= lr[:file_location] %>">
+					      </a>
+					    <% end %>
+
+						<% end %>
+					</div>
+				<% end %>
+
+			  <% unless @presenter.total_viewable_items.blank? %>
+				  <div class="hyc-bugs">
+				    <div class="hyc-item-count">
+				      <b><%= @presenter.total_viewable_items %></b>
+				      <%= 'Item'.pluralize(@presenter.total_viewable_items) %></div>
+
+			      <% unless @presenter.creator.blank? %>
+			        <div class="hyc-created-by">Created by: <%= @presenter.creator.first %></div> <%# Ask Lynette why lname, fname... this is hard to style in inline lists  %>
+			      <% end %>
+
+			      <% unless @presenter.modified_date.blank? %>
+			        <div class="hyc-last-updated">Last Updated: <%= @presenter.modified_date %></div>
+			      <% end %>
+				  </div>
+			  <% end %>
+
+		  </header>
+
+		</div>
+	</div>
+
+	<div class="row hyc-body">
+		<div class="col-md-8 hyc-description">
+			<%= render 'collection_description', presenter: @presenter %>
+		</div>
+    <div class="col-md-4 hyc-metadata">
+      <% unless has_collection_search_parameters? %>
+        <h2>Collection Details</h2>
+        <%= render 'show_descriptions' %>
+      <% end %>
+    </div>
   </div>
-  <div class="col-sm-2">
-    <%= render 'media_display', presenter: @presenter %>
+
+
+  <!-- BL1 -->
```

Comment:
What does this mean? Cruft?

Reply:
Yup. Just little comment marks I had in the code to mark up the Blacklight section. Part of my WIP. The intent is to remove them once I get feedback from Lynette or whoever about how to proceed with that part of the design work.

Reply:
✅